include/titan/options.h

```diff
@@ -249,6 +249,12 @@ struct TitanReadOptions : public ReadOptions {
   // Default: false
   bool key_only{false};
 
+  // If false, it will not abort when failed to get value from blob files and
+  // return an empty value.
+  //
+  // Default: true
+  bool abort_on_failure{true};
```

Comment:
When is this flag set to `false`

Reply:
when test

Reply:
Got it. It's better to comment it that false is only for test.